src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -141,55 +163,74 @@ export class EuiSelectableListItem extends Component<
 
     let state: React.ReactNode;
     let instruction: React.ReactNode;
-    if (allowExclusions && checked === 'on') {
-      state = (
-        <EuiI18n
-          token="euiSelectableListItem.includedOption"
-          default="Selected option."
-        />
-      );
-      instruction = (
-        <EuiI18n
-          token="euiSelectableListItem.includedOptionInstructions"
-          default="To exclude this option, press enter."
-        />
-      );
-    } else if (allowExclusions && checked === 'off') {
-      state = (
-        <EuiI18n
-          token="euiSelectableListItem.excludedOption"
-          default="Excluded option."
-        />
-      );
-      instruction = (
-        <EuiI18n
-          token="euiSelectableListItem.excludedOptionInstructions"
-          default="To uncheck this option, press enter."
-        />
-      );
-    } else if (allowExclusions && !checked) {
-      instruction = (
-        <EuiI18n
-          token="euiSelectableListItem.unckeckedOptionInstructions"
-          default="To select this option, press enter."
-        />
-      );
-    }
-
-    const isChecked = !disabled && typeof checked === 'string';
-    if (!allowExclusions && isChecked) {
-      state = (
-        <EuiI18n
-          token="euiSelectableListItem.checkedOption"
-          default="Checked option."
-        />
-      );
-      instruction = searchable ? (
-        <EuiI18n
-          token="euiSelectableListItem.checkedOptionInstructions"
-          default="To uncheck this option, press enter."
-        />
-      ) : undefined;
+    if (allowExclusions) {
+      switch (checked) {
+        case 'on':
+          state = (
+            <EuiI18n
+              token="euiSelectableListItem.includedOption"
+              default="Checked option."
+            />
```

Comment:
```suggestion
            <EuiI18n
              token="euiSelectableListItem.checkedOption"
              default="Checked option."
            />
```

Since we're making the string the exact same as line 224 down below we should just go ahead and make these the same i18n token/string. Our i18n translators skip having to re-translate the same string that way.

Reply:
Actually, we're repeating a bunch of i18n strings across the board down below. I'd prefer to dry those out into const's I think... any objections if I push up a clean up?

Reply:
I refactored this in https://github.com/elastic/eui/pull/6774/commits/7ee93d4dcbdc9216ffd8c14f5c0f610d1a89450f and found several bugs/missing or confusing SR text that was fixed as a result of cleaning up the branches (namely, `mixed` and non `allowExclusions` options did not have the correct SR text).

I think this should be fixed now - @1Copenut, do you want to do a final pass on _all_ EuiSelectable 

The permutations that generate different screen reader text that need to be tested:

- [Mixed](https://eui.elastic.co/pr_6774/#/forms/selectable#options-can-be-mixed-indeterminate)
- [Excluded](https://eui.elastic.co/pr_6774/#/forms/selectable#options-can-be-excluded)
- [Searchable](https://eui.elastic.co/pr_6774/#/forms/selectable#searchable)
- [Vanilla](https://eui.elastic.co/pr_6774/#/forms/selectable#the-basics)